packages/app-project/pages/_app.js

```diff
@@ -32,17 +32,17 @@ function useStore(initialState) {
 }
 
 export default function MyApp({ Component, pageProps }) {
-  try {
-    const { initialState } = pageProps
-    const store = useStore(initialState)
-    makeInspectable(store)
+  const { initialState } = pageProps
+  const store = useStore(initialState)
```

Comment:
Should the `useStore` hook have a try/catch in case instantiating the store fails? I'm not sure if this whole block of code was wrapped with the try in case that failed. The `user.checkCurrent()` has its own error handling.

Reply:
I’m not sure. We probably want the error logged in Sentry, but we probably want NextJS to generate an error page too. I might play around with this in a separate PR. 

Here, I think the hook inside a block was breaking the Rules of Hooks, but I’m not sure if it’s dangerous in the same way that conditional hooks are dangerous.